docs/reference/configuration.md

```diff
@@ -0,0 +1,22 @@
+# Configuration of gatsby-source-plone
+
+This Plugin provide some options so that you can add more feature to your website.
+
+# How to use
+```Javascript
+  plugins: [
+    {
+      resolve: 'gatsby-source-plone',
+      options: {
+        baseUrl: 'http://localhost:8080/Plone/',
+        token: process.env.TOKEN,
+        logLevel: 'DEBUG',
+        websocketUpdates: true || false,
+      },
+    },
+  ]
+```
+* baseUrl : Points to your Plone CMS Url
+* token : If you provide token as options plugin will be able to fetch our private field data also.
+* logLevel : This will provide additional logging for the plugin. Basically this will provide a debugging environment.
+* websocketUpdates: You can set websocketUpdates as true and false. True will enable the Gatsby Preview for your site. It shows you instant update when you made some changes to Plone CMS.
```

Comment:
Actually there are some options, but these are the "stable" ones.

Could you use table for the parameters and describe them before the example like in https://github.com/gatsbyjs/gatsby/tree/master/themes/gatsby-theme-blog#usage

There may be many styles in the current documentation, but probably it is good to try to move towards other Gatsby projects.

Reply:
ok! I will add a table, explaining all the config options as you suggested :) thanks!

Reply:
I have made changes, please! comment if I am missing something :)

Reply:
Since pr is merged I am resolving this conversation :)